### ISSUE TITLE
npm version lags behind master

### ISSUE THREAD

**Issue by mdressman** [seatgeek/sixpack-js#12]
there have been significant changes Aug 19, 2013 but the version on npm doesn't include any of the updates and doesn't match the documentation.

Can you please bump the package version so users can get the latest?

**Comment by zackkitzmiller** [comment]
Yup! As soon as we're ready to tag a release we'll update the version and publish to npm.

**Comment by zackkitzmiller** [comment]
Should be able to get this out soon

**Comment by mdressman** [comment]
cool, installing straight from github while evaluating in the meantime

**Comment by zackkitzmiller** [comment]
Sounds good. Once we get final review from Zach for the persist PR, I'll publish a new version
